### PR TITLE
IDE page changes

### LINKED ISSUE #2617: Review content for redundancy
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

Pages in this section https://docs.getdbt.com/docs/get-started/getting-started/set-up-dbt-cloud 

#### Content updates 

As part of the Docs team's Guides project:

Review these pages and determine how to consolidate some of the redundant information:

- https://docs.getdbt.com/docs/get-started/dbt-cloud-features
- https://docs.getdbt.com/docs/get-started/dbt-cloud-tips
- https://docs.getdbt.com/docs/get-started/develop-in-the-cloud 

#### Left sidebar updates 

- Update sidebar.js accordingly. Currently, idea is for these pages to live under the "Run dbt projects" section.
- Update links and redirects

### Additional information

_No response_

### LINKED ISSUE #2729: 1. Consolidate ide content in dbt cloud features pages
https://www.notion.so/dbtlabs/Edit-dbt-feature-page-f8c024071d724526881f18e1123d87da

Closes #2617 

### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

1. Move superfluous IDE content from ‘cloud features’ and 'dbt cloud tips' pages → ‘develop in the cloud’ page. 
2. This includes bullets specifying IDE actions, and the IDE features table.
3. Consolidate IDE features table with table on the ‘develop in cloud’ page
4. Link to the ‘Build your first models’ page on ‘Develop in the Cloud’ (under Build, compile, and run projects section)
5. Add redirects for any pages removed

### What part(s) of the page would you like to see updated?

dbt cloud features https://docs.getdbt.com/docs/get-started/dbt-cloud-features
developt in cloud https://docs.getdbt.com/docs/get-started/develop-in-the-cloud
dbt cloud tips https://docs.getdbt.com/docs/get-started/dbt-cloud-tips

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by mirnawong1**
Closes #2729 
Closes #2617 

Linked to #2771 

consolidated features and ide page

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/get-started/develop-in-the-cloud.md [hunk 1]
```diff
@@ -1,66 +1,96 @@
...
 :::info📌
 
 New to dbt? Check out our [Getting Started guide](/docs/get-started/getting-started/overview) to build your first dbt project in the Cloud IDE!
 
 :::
 
-In order to start experiencing the great features of the Cloud IDE, you need to first set up your **Development environment** and **Development credentials.**
+<!-- the link to dbt Cloud environments isn't live just yet, see pr https://github.com/dbt-labs/docs.getdbt.com/pull/2777 -->
+
+In order to start experiencing the great features of the Cloud IDE, you need to first set up a [dbt Cloud environment](docs/collaborate/environments/dbt-cloud-environments#dbt-cloud-environments). Follow the steps in [Development environment](#development-environment) and [Development credentials](#developer-credentials) for more details.
```
Comment (mirnawong1) [inline website/docs/docs/get-started/develop-in-the-cloud.md]: hey @dave-connors-3 @nghi-ly and @matthewshaver - i'm wondering if this bit below (dev environment and credentials) should live in Dave's environments page? it differs in that it breaks down steps to set it up but i think combined with Dave's explanation, may be better suited there?

File: website/docs/docs/get-started/dbt-cloud-features.md [hunk 2]
```diff
@@ -17,9 +17,9 @@ With the Cloud IDE, you can:
 - Generate and view documentation of your dbt project
 - Leverage [git and version-control your code](/docs/collaborate/git/version-control-basics) from your browser with a couple of clicks
 - Create and test Python models:
-    * You can `compile` Python models to see the full function that gets executed in your data platform
-    * You can see Python models in DAG in dbt version 1.3 and higher
-    * You can't preview python models, currently
+    * Compile Python models to see the full function that gets executed in your data platform
+    * See Python models in DAG in dbt version 1.3 and higher
```
Comment (matthewshaver) [inline website/docs/docs/get-started/dbt-cloud-features.md]:
```suggestion
    * See Python models in DAG in version 1.3 and higher
```

File: website/docs/docs/get-started/dbt-cloud-features.md [hunk 3]
```diff
@@ -17,9 +17,9 @@ With the Cloud IDE, you can:
...
+    * Currently you can't preview python models
```
Comment (matthewshaver) [inline website/docs/docs/get-started/dbt-cloud-features.md]:
```suggestion
    * Currently, you can't preview python models
```

Comment (mirnawong1) [reply website/docs/docs/get-started/dbt-cloud-features.md]: omg i couldve sworn i already fixed this with a comma 🤦‍♀️ i must've not saved. thank you @matthewshaver !

**Review by matthewshaver — DISMISSED** [review]

File: website/docs/docs/get-started/develop-in-the-cloud.md [hunk 4]
```diff
@@ -1,72 +1,105 @@
...
-- You must have a git repository set up and your git provider must have `write` access enabled. See [Connecting your GitHub Account](/docs/collaborate/git/connect-github) and [Importing a project by git URL](/docs/collaborate/git/import-a-project-by-git-url) for detailed setup instructions
-- Your dbt project must be connected to a [data platform](/docs/get-started/connect-your-database)
-- You must have a [**development environment** and **development credentials**](#set-up-and-access-the-cloud-ide) set up
+- A [dbt Cloud account](https://cloud.getdbt.com/) and [Developer seat license](/docs/collaborate/manage-access/seats-and-users)
+- A git repository set up and git provider must have `write` access enabled. See [Connecting your GitHub Account](/docs/collaborate/git/connect-github) and [Importing a project by git URL](/docs/collaborate/git/import-a-project-by-git-url) for detailed setup instructions
+- A dbt project connected to a [data platform](/docs/get-started/connect-your-database)
+- Set up a [development environment and development credentials](#set-up-the-cloud-ide)
```
Comment (nghi-ly) [inline website/docs/docs/get-started/develop-in-the-cloud.md]: this link is broken

File: website/docs/docs/get-started/develop-in-the-cloud.md [hunk 5]
```diff
@@ -1,72 +1,105 @@
...
 
 - Unsaved, local code &mdash; Any code you write is automatically available from your browser’s storage. You can see your changes but will lose them if you switch branches or browsers (another device or browser).
 - Saved but uncommitted code &mdash; When you save a file, the data gets stored in your local storage (EFS storage). If you switch branches but don’t *commit* your saved changes, you will lose your changes.
 - Committed code &mdash; This is stored in the branch with your git provider and you can check out other (remote) branches.
 
-
-</td></tr> </table>
-
-## Set up and access the Cloud IDE
+## Set up Environments
```
Comment (nghi-ly) [inline website/docs/docs/get-started/develop-in-the-cloud.md:77]:
```suggestion
## Set up environments
```

Comment (nghi-ly) [reply website/docs/docs/get-started/develop-in-the-cloud.md:77]: sugg: sentence case (not title case)

File: website/docs/docs/get-started/develop-in-the-cloud.md (hunk 4 above)
Comment (mirnawong1) [reply website/docs/docs/get-started/develop-in-the-cloud.md]: thank you! fixed 🙌

File: website/docs/docs/get-started/develop-in-the-cloud.md [hunk 6]
```diff
@@ -1,72 +1,105 @@
...
 
-In order to start experiencing the great features of the Cloud IDE, you need to first set up your **Development environment** and **Development credentials.**
+<!-- the link to dbt Cloud environments isn't live just yet, see pr https://github.com/dbt-labs/docs.getdbt.com/pull/2777 
+
+In order to start experiencing the great features of the Cloud IDE, you need to first set up a [dbt Cloud environment](/docs/collaborate/environments/dbt-cloud-environments#dbt-cloud-environments). 
+-->
+
+In order to start experiencing the great features of the Cloud IDE, you need to first set up a [dbt Cloud environment](/docs/collaborate/environments). Follow the steps in [Development environment](#development-environment) and [Development credentials](#developer-credentials) for more details. 
 
 If you’re new to dbt, you will automatically add this during the project setup. However, if you have an existing dbt Cloud account, you may need to create a development environment and credentials manually to use the Cloud IDE.
```
Comment (nghi-ly) [inline website/docs/docs/get-started/develop-in-the-cloud.md:91]: thanks, @mirnawong1 ! happy to take a look at this for you! this ^ sentence and the one above it are almost saying the same thing. suggest reworking the two paragraphs for clarity and to limit redundancy

Comment (mirnawong1) [reply website/docs/docs/get-started/develop-in-the-cloud.md:91]: good sugg! this is done now

File: website/docs/docs/get-started/develop-in-the-cloud.md (hunk 1 above)
Comment (nghi-ly) [reply website/docs/docs/get-started/develop-in-the-cloud.md]: @mirnawong1 : good idea! just need to make sure the transition to these sections on dave's page are clear

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**